File: cielo24_Java/cielo24/src/cielo24/json/Segment.java
package cielo24.json;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class Segment extends JsonBase {
	@SerializedName("sequences")
	public List<Sequence> sequences;
	@SerializedName("speaker_change")
	public boolean speakerChange;
	@SerializedName("speaker_id")
	public boolean speakerId;
	@SerializedName("interpolated")
	public boolean interpolated;
	@SerializedName("start_time")
	public int startTime;
	@SerializedName("end_time")
	public int endTime;
}

File: cielo24_.NET/Cielo24/JSON/ElementList/Segment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Newtonsoft.Json;

namespace Cielo24.JSON.ElementList
{
    public class Segment : JsonBase
    {
        [JsonProperty("sequences")]
        public List<Sequence> Sequences { get; set; }
        [JsonProperty("speaker_change")]
        public bool SpeakerChange { get; set; }
        [JsonProperty("speaker_id")]
        public bool SpeakerId { get; set; }
        [JsonProperty("interpolated")]
        public bool Interpolated { get; set; }
        [JsonProperty("start_time")]
        public int StartTime { get; set; }
        [JsonProperty("end_time")]
        public int EndTime { get; set; }
    }
}
